docs/conversion/sakai_21_oracle_conversion.sql

```diff
@@ -215,8 +215,9 @@ ALTER TABLE SSQ_SITETYPE_QUESTIONS MODIFY SITE_TYPE VARCHAR2(255 CHAR) NOT NULL;
 -- END SAK-45184
 
 -- START SAK-42371
-ALTER TABLE rbc_evaluation ADD status NUMBER(10,0) NOT NULL;
-UPDATE rbc_evaluation SET status = 2;
+ALTER TABLE rbc_evaluation ADD status NUMBER(1);
```

Comment:
@vacercon

Why number(1) instead of number(10) ?

Reply:
Makes sense to me!

Reply:
Thanks Jesus, i forget NOT NULL. 
I do not understand ¿Original sentence set 2 as the value when is not defined?
UPDATE rbc_evaluation SET status = 2;

I think it's better
UPDATE rbc_evaluation SET status = 2 WHERE status is null;
To avoid possible problems